dss/__init__.py

```diff
@@ -17,10 +17,14 @@
 
 logging.basicConfig(level=logging.DEBUG)
 
+app = None
```

Comment:
Good point. I didn't know about `flask.current_app`. That works better as a global accessor. I'll delete this one.